src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/ContentDbUtils.cs

```diff
@@ -1,14 +1,16 @@
 using System;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
 using Microsoft.EntityFrameworkCore;
 
-namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Database
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils
```

Comment:
Not sure I'm following the reason for moving this to `Common.Tests`, shouldn't it be in `Content.Mode.Tests`?

Same applies to moving `StatisticsDbUtils`. By moving this to `Common.Tests` any test project wanting to use `Common.Tests` as a dependency now has a dependency on the entire Content and Statistics models and all of the SqlServer dependencies.